code/modules/admin/verbs/atmosdebug.dm

```diff
@@ -12,23 +12,23 @@
 
 	usr << "Checking for disconnected pipes..."
 	//all plumbing - yes, some things might get stated twice, doesn't matter.
-	for (var/obj/machinery/atmospherics/plumbing in world)
+	for (var/obj/machinery/atmospherics/plumbing in machines)

```

Comment:
I don't believe all atmos devices are left in this list; this is a map-debug command anyways.